app/javascript/react/components/Navbar/Navbar.tsx

```diff
@@ -30,68 +31,97 @@ const NavItem = ({
 
 const Navbar = () => {
   const [navMenuVisible, setNavMenuVisible] = useState(false);
+  const { t } = useTranslation();
 
   return (
     <S.Header>
-      <a href="https://www.habitatmap.org/" aria-label="AirCasting page">
-        <S.AircastingLogo alt="Aircasting logo" src={logo} />
+      <a
+        href={urls.habitatMap}
+        aria-label={t("navbar.sections.aircastingPage")}
+      >
+        <S.AircastingLogo alt={t("navbar.altLogo")} src={logo} />
       </a>
       <S.Container>
-        <S.BuyCTA href={urls.airbeamBuy}>get airbeam</S.BuyCTA>
+        <S.BuyCTA href={urls.airbeamBuyNow}>
+          {t("navbar.sections.getAirbeam")}
+        </S.BuyCTA>
```

Comment:
How about this version? :) 
```suggestion
        <S.BuyCTA href={urls.getAirbeam}>
          {t("navbar.sections.getAirbeam")}
        </S.BuyCTA>
```
What I meant before. :)

Reply:
Oh, you have probably just declared it once and then using everywhere using its name- which makes perfect sense and I don't. :D In such case ignore me. 🙃